provision-mysql.sh

```diff
@@ -49,7 +49,7 @@ configure_mysql()
 		tell_status "installing $_my_cnf"
 		tee -a "$_my_cnf" <<EO_MY_CNF
 [mysqld]
-innodb_doublewrite = off
+#innodb_doublewrite = off
```

Comment:
I had a corrupt DBs after restoring an image of my VPS, took me a lot of effort to get the DATA back.

So I'm not sure if this setting creates a corrupt DB, but to be on the safe side I disabled it.

See here the last comment:
https://dba.stackexchange.com/questions/86636/when-is-it-safe-to-disable-innodb-doublewrite-buffering

> It is true that, when using data=journal, EXT4 will guarantee that application data are double-written to both the journal and the main filesystem. However _if the application crashes during a write_ the filesystem will receive incomplete data, yet it does not know anything about that.

> Example:
> a C program is writing a large enough (eg: 128KB) buffer using a write call. Suddenly, it is killed just when the write call is transferring the application buffer’s content into OS pagecache. After some time (or by issuing a sync) the filesystem receive the partial data and double-write them. The key point here is that, while data are consistent from filesystem view, they are _not_ consistent from the application view.

> Using only small buffer (eg: 16KB) will rarely trigger the problem, but using bigger buffer surely will (the example with 128 KB buffer is quite telling).

> So: do NOT disable InnoDB doublewrite, unless you _really_ know not only what you are doing, but the OS-level implication in doing that.